apps/api/src/app/subscribers/subscribers.controller.ts

```diff
@@ -223,6 +228,29 @@ export class SubscribersController {
     );
   }
 
+  @Delete('/:subscriberId/credentials/:providerId')
+  @ExternalApiAccessible()
+  @UseGuards(JwtAuthGuard)
+  @ApiResponse(SubscriberResponseDto)
+  @ApiOperation({
+    summary: 'Delete subscriber credentials by providerId',
+    description: 'Delete subscriber credentials such as slack and expo tokens.',
+  })
```

Comment:
usually delete endpoints just return the 200 or 204
like: `@HttpCode(HttpStatus.OK) or @HttpCode(HttpStatus.NO_CONTENT)`

Reply:
I personally push to always return the 204 and empty body as you should not need anything else besides the confirmation the operation was right.